docs/faq.pug

```diff
@@ -112,7 +112,7 @@ block content
     // Promise based alternative. `init()` returns a promise that resolves
     // when the indexes have finished building successfully. The `init()`
     // function is idempotent, so don't worry about triggering an index rebuild.
-    Model.init().then(function() {
+    Model.init().then(function(err) {
       assert.ifError(err);
```

Comment:
Welcome @tigransimonyan 

The function passed to `.then(...)` is only invoked in case of success, so in this case, we should remove `assert.ifError(err);` and the parameter you just added.

Reply:
Done! :D

Reply:
Thanks 👍